bordel

```diff
@@ -84,6 +84,9 @@ fi
 if [ -d "${TOP}/${BUILD_DIR}" -a "${command}" != "config" ]; then
 	pushd "${TOP}/${BUILD_DIR}" >/dev/null
 fi
+if [[ "${command}" == "build" ]]; then
```

Comment:
I would suggest moving this block to be at the beginning of `build_main` in cmds/build. It would be equivalent and drops the test.

Reply:
Done, not sure why I didn't do that in the first place lol. Also note that I add a certs generation step in the README; I didn't realize it was a mandatory step for some recipes.

Reply:
Great, thanks!